oxidation/libparsec/crates/platform_http_proxy/src/lib.rs

```diff
@@ -109,6 +111,9 @@ mod not_wasm32 {
     }
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+pub use not_wasm32::{HTTPS_PROXY, HTTP_PROXY};
```

Comment:
the names to use in platform are `native` and `web`

(don't use `wasm32` given we specifically target web i.e. we consider we have access to the browser API)